Dockerfile

```diff
@@ -1,7 +1,17 @@
 FROM nvidia/cuda:10.1-cudnn7-devel
 # To use this Dockerfile:
-# 1. `nvidia-docker build -t detectron2:v0 .`
-# 2. `nvidia-docker run -it --name detectron2 detectron2:v0`
```

Comment:
It seems the new feature is supported only after docker 19.03, which is still quite new. It would be better to keep both old and new instructions.